quupe-RN/js/components/UpdateProfileForm/UpdateProfileForm.js

```diff
@@ -0,0 +1,192 @@
+                                                                Accept:
+                                                                    'application/json',
+                                                                'Content-Type':
+                                                                    'multipart/form-data'
+                                                            }
+                                                        };
+                                                        return fetch(
+                                                            'https://api.graph.cool/file/v1/cjidp1w9z1cn30149s8454eyu',
+                                                            options
+                                                        )
+                                                            .then(response => {
+                                                                console.log(response);
+                                                                return response.json();
+                                                            })
+                                                            .then(image => {
+                                                                this.updateProfileImage(image.url);
+                                                                console.log(this.props
+                                                                    .profileimage);
+                                                                return image;
+                                                            })
+                                                            .catch(error =>
+                                                                console.error('Error uploading image'));
+                                                    }
+                                                }
+                                            );
+                                        }}
+                                    >
+                                        <Text>Upload Image</Text>
+                                    </TouchableHighlight>
+
+                                    <TouchableOpacity
+                                        disabled={invalid || pristine}
+                                    >
+                                        <Text onPress={handleSubmit}>
+                                            Update Profile
+                                        </Text>
+                                    </TouchableOpacity>
+                                </View>
+                            )}
+                        />
+                    )}
+                </Mutation>
+            </ScrollView>
+        );
+    }
+}
+BioForm.propTypes = {
+    dispatch: PropTypes.func.isRequired
+};
+
+export default graphql(updateUser, { name: 'updateUser' })(connect(state => ({
+    title: state.updateProfile.fullname,
```

Comment:
If you are connecting your component to Redux, you will be receiving state from Redux as props. So please add prop type validation for all the props you receive in your component.